src/App.vue

```diff
@@ -375,6 +377,12 @@ export default {
 		} else if (BrowserStorage.getItem('sidebarOpen') === 'true') {
 			this.$store.dispatch('showSidebar')
 		}
+		if (this.isMobile) {
+			await this.$nextTick()
+			emit('toggle-navigation', {
+				open: true,
```

Comment:
innit `false` here??

Reply:
no, we want the chatview to load on mobile automatically.

Reply:
But not when you opened a URL of a conversatino already

Reply:
fixed